packages/driver/cypress/integration/e2e/multi-domain/multi_domain_validation_specs.ts

```diff
@@ -1,6 +1,6 @@
-describe('multi-domain', () => {
+describe('cy.origin', () => {
   describe('successes', () => {
-    it('succeeds on a localhost domain name', () => {
+    it('succeeds on a localhost origin name', () => {
```

Comment:
Since this is not a complete origin:
```suggestion
    it('succeeds on a localhost domain name', () => {
```